O365DataRetriever.ps1

```diff
@@ -765,11 +768,13 @@ $ConnectButton.Add_Click( {
         Import-PSSession $SfBOsession -AllowClobber | Out-Null
 
         #Connect to SPO
-		Write-Host "Connecting to SharePoint Online..." -ForegroundColor Cyan
-        Connect-SPOService -Url https://$SPOTenant-admin.sharepoint.com -Credential $creds
+        Write-Host "Connecting to SharePoint Online..." -ForegroundColor Cyan
+        $creds = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList ($AdminUserName, $(ConvertTo-SecureString -String $AdminPwdTextbox.Password -AsPlainText -Force))
+        Connect-SPOService -Credential $creds -Url "https://$SPOTenant-admin.sharepoint.com"
 
         #Connect to the Compliance Center
-		Write-Host "Connecting to the Compliance Center..." -ForegroundColor Cyan
+        Write-Host "Connecting to the Compliance Center..." -ForegroundColor Cyan
+        $creds = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList ($AdminUserName, $(ConvertTo-SecureString -String $AdminPwdTextbox.Password -AsPlainText -Force))
```

Comment:
Needs more investigation with VSCode or more issues related to this before changing it.